pos_customer_required/demo/pos_config.yml

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Please use header like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py